run_YACHT.py

```diff
@@ -32,7 +33,8 @@
                                                            'Each value should be between 0 and 1, with 0 being the most sensitive (and least '
                                                            'precise) and 1 being the most precise (and least sensitive).', 
                                                            required=False, default=[1, 0.5, 0.1, 0.05, 0.01])
-    parser.add_argument('--out_filename', help='Full path of output filename', required=False, default='result.xlsx')
+    parser.add_argument('--out_filename', help='output excel filename', required=False, default='result.xlsx')
```

Comment:
In general, it's best to have just on "out" parameter, which was the motivation behind my combining these. Without it, you can't take advantage of linux constructs like `../out_file.ext`. Can you revert to the single output format?

Reply:
Hi @dkoslicki, please don't worry about the path format, it works for all different Linux constructs because it has [this](https://github.com/KoslickiLab/YACHT/blob/e98f3e5bf854af8a7ee18ddb597037d6187d80eb/run_YACHT.py#L49) line. Actually, the default `--out_filename` is result.xlsx, you don't need to change it for most of time. You just need to specify its `--outdir`.

Reply:
It’s not good practice to choose the name of the output file for the user. [Source1](https://clig.dev/#arguments-and-flags:~:text=Actions%20crossing%20the%20boundary%20of%20the%20program%E2%80%99s%20internal%20world%20should%20usually%20be%20explicit.%20This%20includes%20things%20like%3A). Source 2: I recall using tools that auto-name the output and have had data get lost since the program decided to overwrite my `output.ext` file without telling me about it. 
Please change back to the convention of a single output file name, that can be a full absolute path, relative path, etc.

Reply:
ok, now we only use `--out` to specify the output in `run_YACHT.py`. Please see those changes in https://github.com/KoslickiLab/YACHT/pull/43/commits/214d2cf997dc1725d5f40bbf74f9e5d81bace48c.